pkg/apis/pipeline/v1beta1/pipeline_validation_test.go

```diff
@@ -2289,6 +2289,8 @@ func TestPipelineTasksExecutionStatus(t *testing.T) {
 			TaskRef: &TaskRef{Name: "bar-task"},
 			Params: []Param{{
 				Name: "foo-status", Value: ArrayOrString{Type: ParamTypeString, StringVal: "$(tasks.foo.status)"},
+			}, {
+				Name: "tasks-status", Value: ArrayOrString{Type: ParamTypeString, StringVal: "$(tasks.status)"},
```

Comment:
please add a similar test with a finally task accessing the aggregate status of tasks in whenexpressions

Reply:
yup, have added when expression accessing aggregate status of the tasks